Controllers/EmployeeController.cs

```diff
@@ -16,12 +16,105 @@ namespace TodoApi.Controllers
     public class EmployeeController : ControllerBase
     {
         private readonly ApiDbContext _context;
+        private readonly string connectionString = "Server=myserver;Database=mydb;User Id=admin;Password=password;";
 
 
         public EmployeeController(ApiDbContext context)
         {
             _context = context;
         }
+        
+       [HttpGet]
+        public async Task<IActionResult> GetEmployeeBlocking()
+        {
+            var employee = _context.Employees.FindAsync(1).Result; // Blocking async call
+            return Ok(employee);
+        }
+
+        [HttpGet]
+        public async Task<IActionResult> SlowGetEmployee()
+        {
+            Thread.Sleep(5000); // Blocking thread in async code
+            var employee = await _context.Employees.FindAsync(1);
+            return Ok(employee);
+        }
+
+        [HttpGet]
+        public async Task<IActionResult> GetEmployeeNoDispose()
+        {
```

Comment:
Avoid using raw SQL queries directly in the controller to prevent SQL injection attacks. Use parameterized queries or ORM methods instead.